services/swagger/swagger.tester.js

```diff
@@ -21,6 +24,27 @@ t.create('Valid')
     color: 'brightgreen',
   })
 
```

Comment:
Let's remove any of the mocked tests that are duplicative of the real/live tests. You acn also remove the `Live` prefix from the test names, as our test runner automatically denotes whether a test is mocked or live in the output

Reply:
I removed all mocked ones besides the invalid one